.gitlab-ci.yml

```
@@ -4,10 +4,11 @@ stages:
   
 variables:
   DATADOG_AGENT_WINBUILDIMAGES: v5996510-a9e6a7d
+  DRIVER_S3_BUCKET: dd-windowsfilter
     
 driver_build:
   only:
-    - web
+    - master
```

For future proofing, should we add `main` here too?
```suggestion
    - master
    - main
```

---

Done in 1ad8ac42a336fecd7aca60a84fe6b98aca0109d7